python/kat/harness.py

```diff
@@ -203,7 +203,7 @@ def sanitize(obj):
 
 
 def abstract_test(cls: type):
-    cls.abstract_test = True
+    cls.abstract_test = True  # type: ignore
```

Comment:
Having `mypy` check these now is great! Of course that means we should actually fix the type errors rather than ignoring them. 🙂 

I'd be OK with that being a PR right after this one, if need be.

Reply:
:+1: